client/public/materializeCSS/materialize.css

```diff
@@ -3,7 +3,15 @@
  * Copyright 2014-2017 Materialize
  * MIT License (https://raw.githubusercontent.com/Dogfalo/materialize/master/LICENSE)
  */
-.materialize-red {
+
+ .loginbox, .signupform{
+    background: white;
+    opacity: .8;
+    padding: 20px;
+    margin-top: 40% !important;
```

Comment:
why do we need !important? I'm assuming it's needed in order for the styling to be what you guys want, but then again if you architected all of your css ideally you shouldn't have to use important that much.

Reply:
Agree that's a red flag and usually is not needed.